app/scripts/controllers/users/create.js

```diff
@@ -0,0 +1,86 @@
+'use strict';
+
+/**
+ * @ngdoc function
+ * @name flightNodeDemo.controller:UserCreateController
+ * @description
+ * # UserCreateController
+ * Controller for the create user page.
+ */
+angular.module('flightNodeDemo')
+	.controller('UserCreateController',
+		['$scope', '$http', '$log', '$location', 'messenger', 'roleProxy',
+			function ($scope, $http, $log, $location, messenger, roleProxy) {
+
+				$scope.loading = true;
+				$scope.data = {};
+
+				var jwt = JSON.parse(localStorage.getItem('jwt'));
+
```

Comment:
Added the following lines, `roleProxy.getAll`, yesterday. This calls the /v1/role endpoint to retrieve the available roles and bind them to a listbox.